src/main/java/io/github/anthorx/parquet/sql/write/NullTypeConsumer.java

```diff
@@ -0,0 +1,90 @@
+package io.github.anthorx.parquet.sql.write;
+
+import io.github.anthorx.parquet.sql.record.ReadRecordConsumer;
+import org.apache.parquet.schema.LogicalTypeAnnotation;
+import org.apache.parquet.schema.PrimitiveType;
+import org.apache.parquet.schema.Type;
+
+import java.sql.Types;
+import java.util.Optional;
+import java.util.function.Consumer;
+
+/**
+ * create the Consumer that calls ReadRecordConsumer.setNull with the proper type
+ */
+public class NullTypeConsumer {
```

Comment:
The class name lie, it's not a Consumer as we define it into the library (Java Consumer).
Find another name !